packages/homepage/src/components/Toast/index.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useState, useEffect } from 'react';
+import { Link } from 'gatsby';
+import { AnimatePresence } from 'framer-motion';
+
+import { ToastContainer } from './elements';
+
+const Privacy = () => {
+  const key = 'ACCEPTED_TERMS_CODESANDBOX_HOME';
+  const [show, setShow] = useState(false);
```

Comment:
You can use a callback function to set initial value.

```suggestion
  const [show, setShow] = useState(() => !window.localStorage.getItem(key));
```